code/tailer/datastripe.q

```diff
@@ -151,17 +151,26 @@ savetables:{[dir;tablename]
     .gc.run[];
     };
 
-savealltables:{[dir]
+savealltablesoverperiod:{[dir;lasttime]
     /- function takes the tailer hdb directory handle and a timestamp
     /- saves each table up to given period to their respective partitions
-    savetables[dir;]each .wdb.tablelist[];
-	
-    /- delete data that has been saved
-    @[`.;;0#] each .wdb.tablelist[];
-	
+    /- totals calculates the row count of tables in .wdb.tablelist[]
+    totals:{count value x}each .wdb.tablelist[];
+    /- log and return from function early if no table has crossed threshold
+    if[all totals<.wdb.numrows;
+        .lg.o[`save;"No tables above threshold, no tables saved"];
+        :();
+        ];
```

Comment:
```suggestion
    ];
```

IMO the closing `]` of a multi-line `if` should be aligned with the `i` in opening line, just makes it clearer when scanning the file/function quickly